src/app/app-routing.module.ts

```diff
@@ -1,8 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component'
+import { HomeComponent } from './home/home.component'
```

Comment:
The changes are looking good.  currently this doesn't build because you forgot to remove HomeComponent from here.  It is always a good idea to serve and view the app locally before committing and pushing up changes